ground/src/main/res/values/strings.xml

```diff
@@ -151,14 +151,13 @@
   <string name="about">About</string>
   <string name="terms_of_service">Terms of service</string>
   <string name="about_ground">Ground is an open-source community project built by Google and FAO under the Forest Data Partnership with help from SIG, Ecam, and open-source community contributors. \n\nIt is licensed under the <annotation type="apache_license">Apache License, Version 2.0</annotation>. \n\nView other <annotation type="all_licenses">Licenses</annotation></string>
-  <string name="failed">Failed</string>
-  <string name="synced">Synced</string>
-  <string name="syncing">Syncing</string>
-  <string name="upload_pending">Upload pending</string>
-  <string name="data_synced_media_pending">Data synced\n(media pending)</string>
-  <string name="data_synced_media_pending_retry">Data synced\n(media pending retry)</string>
-  <string name="media_syncing">Media syncing</string>
-  <string name="data_sync_status">Data sync status</string>
+  <string name="retry_pending">Retry pending</string>
+  <string name="upload_complete">Complete</string>
+  <string name="uploading">Uploading</string>
+  <string name="upload_pending">Pending</string>
+  <string name="photos_pending">Photos pending</string>
+  <string name="photos_pending_retry">Photos pending retry</string>
+  <string name="uploading_photos">Uploading photos</string>
```

Comment:
@amegantz Could you PTAL at these to see if they work?

@jo-spek Once @amegantz approves, do you have someone who could verify the French, Spanish, and Portuguese translations I added to other `strings.xml` files?

Reply:
Hi @gino-m, yes, as agreed upon in the stand-up, I'll regularly keep an eye on these! Please tag me whenever you add something to strings.xml.

Reply:
We're doing a small redesign of this screen so I'll pause this PR for now. Will def keep you posted on updated/new strings!